packages/community-cli-plugin/src/utils/loadMetroConfig.js

```diff
@@ -107,10 +124,16 @@ This warning will be removed in future (https://github.com/facebook/metro/issues
     }
   }
 
-  const config = await loadConfig({
-    cwd,
-    ...options,
-  });
+  const config = await loadConfig(
+    {
+      cwd,
+      ...options,
+    },
+    {
+      // Enables users to explicitly specify watchFolders
+      watchFolders: undefined,
+    },
+  );
```

Comment:
This would need an update once https://github.com/facebook/metro/pull/1206 is merged, released and pulled in.

```suggestion
  const config = await loadConfig(
    {
      cwd,
      ...options,
    },
    {
      // Enables users to explicitly specify watchFolders
      watchFolders: undefined,
    },
    false
  );
```